models/project/board.go

```diff
@@ -55,6 +55,7 @@ type Board struct {
 	Default bool   `xorm:"NOT NULL DEFAULT false"` // issues not assigned to a specific board will be assigned to this board
 	Sorting int8   `xorm:"NOT NULL DEFAULT 0"`
 	Color   string `xorm:"VARCHAR(7)"`
+	LabelID int64  `xorm:"DEFAULT 0"`
```

Comment:
Only one label is allowed?

Reply:
And migrations are necessary.

Reply:
We did not see the relevance to assign more than 1 label on column change. see just above where I explained it :)
https://github.com/go-gitea/gitea/pull/28745#issuecomment-1888999589